scss/style.scss

```diff
@@ -20,13 +20,14 @@
 
 //Custom style
 @import "../../../../../profiles/ding2/themes/culture/scss/components/node/event";
+@import "../../../../../profiles/ding2/themes/culture/scss/components/node/campaign";
 @import "../../../../../profiles/ding2/themes/culture/scss/components/node/library";
 @import "../../../../../profiles/ding2/themes/culture/scss/components/node/page";
 @import "../../../../../profiles/ding2/themes/culture/scss/components/node/news";
 @import "../../../../../profiles/ding2/themes/culture/scss/components/_search.scss";
 @import "../../../../../profiles/ding2/themes/culture/scss/components/_footer.scss";
 @import "../../../../../profiles/ding2/themes/culture/scss/components/_header.scss";
-@import "../../../../../profiles/ding2/themes/culture/scss/components/modules/_ding_nodelist.scss";
+@import "../../../../../profiles/ding2/themes/culture/scss/components/modules/ding_nodelist.scss";
```

Comment:
@mihaelad Did you add this manually? This file does not exist. Do you mean `_nodelist.scss ` ?

Reply:
yes this is name in base theme